vaultclient/src/test/java/com/scality/vaultclient/utils/VaultServicesTestConstants.java

```diff
@@ -3,15 +3,16 @@
 public final class VaultServicesTestConstants {
 
     public static final String DEFAULT_ACCOUNT_ID = "123123123123";
-
     public static final String DEFAULT_ARN_STR = "\"arn:aws:iam::123123123123:/";
-
     public static final String DEFAULT_CANONICAL_ID = "31e38bcfda3ab1887587669ee25a348cc89e6e2e87dc38088289b1b3c5329b30";
-
     public static final String DEFAULT_ACCOUNT_NAME = "Account5425";
-
     public static final String DEFAULT_EMAIL_ADDR = "xyz@scality.com";
 
+    public static final String DEFAULT_USER_ID = "456456456456";
+    public static final String DEFAULT_USER_NAME = "User456";
+    public static final String DEFAULT_USER_ARN_STR = "\"arn:aws:iam::123123123123:user/User456";
```

Comment:
Not sure to understand why we have an escaped double quote at the beginning? Is it because the returned ARN also contains a double quote?

Reply:
I think it's a typo of[ this line](https://github.com/scality/vaultclient-java/pull/22/files/9985a3bd0debfa572fe53aeac17571a2a92ba70c#diff-e997f86abf36810095e77b17eb89e3ed267e35cd08e93ef5c021e91333d29108R6)

and I copied from that line 😅  I will remove them